compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -189,12 +190,17 @@ static void addBufferizePasses(OpPassManager &funcPassManager) {
 }
 
 static void tileAndDistributeToWorkgroup(
-    OpPassManager &funcPassManager,
+    OpPassManager &funcPassManager, bool useForall = false,
```

Comment:
I think one of our goals is to use forall for all the cases. Thus, I'd suggest to not have the default value here. Instead, we explicitly add the value for each call. This makes the tracking easier. We can easily track if all the usages are `true` or not. We can remove the option when it happens.